content/blog/gen-ai-for-pm-series-part-4-multi-agent-systems-for-multi-stakeholder-projects.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Multi-agent systems for multi-stakeholder projects
+date: 2025-08-30T06:50:31.006Z
+author: Dinesh R Singh, Nisha Rajput, Varsha Shekhawat
+authorimage: /img/blog-image.jpeg
+disable: false
+tags:
+  - Gen AI
+  - AI agent
+  - Agentic AI
+  - AI project manager
+  - Scrum master
+  - Multi-agent
+  - "Multi-stakeholder "
+---
+<style>
+
+li {
+
+   font-size: 27px;
+
+   line-height: 33px;
+
+   max-width: none;
+
+}
+
+</style>
+
+\*Part 4 of our **Gen AI for PM series***
```

Comment:
Please remove the "\\" character in front of *Part 4 of our **Gen AI for PM series***